src/main/java/hudson/MockExtensionLists.java

```diff
@@ -1,52 +1,65 @@
+/**
+ * This mocks a few different
+ */
```

Comment:
This seems like a net loss of description in the Javadoc.  The new comment is incomplete and the old comment described the features of the class.
```suggestion
```